lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,37 @@
+require 'httparty'
+
+class EdamamApiWrapper
+  class EdamamError < StandardError; end
+
+  APP_ID = ENV["EDAMAM_APP_ID"]
+  APP_KEY = ENV["EDAMAM_APP_KEY"]
+  BASE_URL = "https://api.edamam.com/search?"
+  BASE_URI = "http://www.edamam.com/ontologies/edamam.owl#recipe_"
```

Comment:
The variable names `BASE_URL` and `BASE_URI` are very similar - I would be worried about typoing one for the other. Could you choose something more visually distinct (maybe `BASE_RECIPE_ID`)?